doc/en/usage.rst

```diff
@@ -182,6 +182,47 @@ This will add an extra property ``example_key="1"`` to the generated
       </properties>
     </testcase>
 
+LogXML: add_global_property
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.10
+
+If you want to add a properties node in the testsuite level, which may contains properties that are relevant
+to all testcases you can use ``LogXML.add_global_properties``
+Note that this feature will be available on v2.10.1.
```

Comment:
You may remove this note, it is not necessary because of the `versionadded` directive above.